Sources/Apollo/InMemoryNormalizedCache.swift

```diff
@@ -4,16 +4,14 @@ public final class InMemoryNormalizedCache: NormalizedCache {
   private var records: RecordSet
   private let recordsLock = NSRecursiveLock()
 
-  public init(records: RecordSet = RecordSet()) {
+  public init(records: RecordSet = .init()) {
```

Comment:
reminder: Please do not use `.init`, please use `RecordSet()`

Reply:
Sorry! style habit...